cupy/cusolver.py

```diff
@@ -190,17 +200,19 @@ def _gesvdj_batched(a, full_matrices, compute_uv, overwrite_a):
     u = _cupy.empty((batch_size, m, ldu), dtype=a.dtype).swapaxes(-2, -1)
     v = _cupy.empty((batch_size, n, ldv), dtype=a.dtype).swapaxes(-2, -1)
     params = _cusolver.createGesvdjInfo()
-    lwork = helper(handle, jobz, m, n, a.data.ptr, lda, s.data.ptr,
+    lwork = helper(handle, jobz, m, n, ap.data.ptr, lda, s.data.ptr,
                    u.data.ptr, ldu, v.data.ptr, ldv, params, batch_size)
     work = _cupy.empty(lwork, dtype=a.dtype)
-    info = _cupy.empty(1, dtype=_numpy.int32)
-    solver(handle, jobz, m, n, a.data.ptr, lda, s.data.ptr,
+    info = _cupy.empty(batch_size, dtype=_numpy.int32)
```

Comment:
This is a bug: both cuSOLVER and rocSOLVER need the `info` array to be of the batch size. I thought we fixed it earlier...?

Reply:
Backported this fix to v8 (#4747).